index.html

```diff
@@ -282,6 +282,10 @@ <h2>The Club</h2>
           <td><a href="https://github.com/sisshiki1969/monoruby">monochrome</a></td>
           <td>a Ruby JIT compiler, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://github.com/IAmTheNerdNextDoor/arikoto">Arikoto</a></td>
+          <td>an x86_64 Operating System inspired by ToaruOS, for fun</td>
```

Comment:
```suggestion
          <td>an Operating System inspired by ToaruOS, for fun</td>
```

Sorry to be a pain, but can you make this 5 words or less (not count "an" and "for fun"), according to the PR template? I made a suggestion, but you can reword how you see fit!

Reply:
Sure! People will click on it and see that it's x86_64 anyway, so not much point having x86_64 in there. In the future, it may be ported to other platforms which would then make this incorrect, so it's a change that I'm fine with.